pkg/admissioncontroller/webhook/auth/seed/authorizer.go

```diff
@@ -179,6 +181,8 @@ func (a *authorizer) Authorize(_ context.Context, attrs auth.Attributes) (auth.D
 			return a.authorizeRead(requestLog, seedName, graph.VertexTypeProject, attrs)
 		case secretBindingResource:
 			return a.authorizeRead(requestLog, seedName, graph.VertexTypeSecretBinding, attrs)
+		case credentialsBindingResource:
+			return a.authorizeRead(requestLog, seedName, graph.VertexTypeCredentialsBinding, attrs)
```

Comment:
Could you also adjust the documentation please?
https://github.com/gardener/gardener/blob/master/docs/deployment/gardenlet_api_access.md

Reply:
Thanks for noticing! I somehow forgot about that. PTAL, comments should be addressed now.